src/platform_impl/web/web_sys/canvas/pointer_handler.rs

```diff
@@ -79,9 +79,7 @@ impl PointerHandler {
                     event::mouse_button(&event),
                     event::mouse_modifiers(&event),
                 );
-                canvas
-                    .set_pointer_capture(event.pointer_id())
-                    .expect("Failed to set pointer capture");
+                let _e = canvas.set_pointer_capture(event.pointer_id());
```

Comment:
Nit: Document why we're swallowing the error. Something similar to what @cwfitzgerald wrote is fine:

> ... the error will occur every single time you click the mouse when the cursor is grabbed. [...] there's [not] a situation where this could fail that we really care that it fails.

Reply:
Are there any other cases where we expect calling `setPointerCapture` to fail?

Reply:
I do not know to be honest, so if anyone else has any idea, feel free to suggest them

Reply:
So, I looked up [the spec](https://w3c.github.io/pointerevents/#setting-pointer-capture), and it seems like the call will fail if there is no [active pointer](https://w3c.github.io/pointerevents/#dfn-active-pointer) (and it will also fail "If the element is not connected", but that's a weird situation to be in, I think).